Workflows/Operation/01_ManageWorkflows.md

```diff
@@ -0,0 +1,313 @@
+
+4. Click the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button next to the port you want to connect the place with.    
+  A new place with a connection to the selected port is displayed.
+
+5. Click the new place, drag it over the place whose connection you have deleted and drop it.     
+  The places are merged and the new connection is created.
+
+
+#### Add an action
+
+1. Navigate to the part in the workflow where you want to add an action.  
+
+2. Click the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button next to the output port to which you want to connect a new action.     
+A new place connected to the selected output port is displayed.
+
+  > [Info] If a place to the output port already exists, for instance because it is mandatory, you can skip the step **2**.                                                                 .
+
+  ![New place](/Assets/Screenshots/Workflows/Workflows/NewPlace.png "[New place]")
+
+3. Click the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button right to the new place.    
+  A window to search for an action is displayed.
+
+  ![Search action](/Assets/Screenshots/Workflows/Workflows/SearchAction.png "[Search action]")
+
+4. Click the *search for an action* field and enter the name of the action or a keyword you are searching for.
+
+  > [Info] The list of actions is filtered for your keyword as you type.
+
+5. Click the action you want to add in the list of actions.    
+  The action is added to the workflow and connected to the selected place.
+
+  ![Add action](/Assets/Screenshots/Workflows/Workflows/AddAction.png "[Add action]")
+
+
+#### Edit an action
+
+1. Click the action you want to edit.  
+  The action is highlighted and its settings are displayed in the settings side bar on the right.
+
+  ![Action](/Assets/Screenshots/Workflows/Workflows/Action.png "[Action]")
+
+  > [Info] The settings displayed in the settings side bar may differ depending on the respective action. The sections *Configuration* and *Static Inputs* are only available for certain actions.     
+
+
+2. Edit the settings field in the settings side bar. You can change the label, the queue type, the task event, the configuration and the static inputs as described below:
+
+  + Click the *Label* field and edit the name of the action displayed in the workflow editor.
+
+  + Click the *Queue Type* drop-down list and select the appropriate queue type for the action. All available queue types are displayed in the drop-down list. By default the *Default* queue type is selected. You can define a different queue type for each action within the workflow.
+
+  + Click the *Task Event* drop-down list and select an event connected to the action. The event is triggered and the corresponding task is assigned when the action is failed within a process. All available task events are displayed in the drop-down list. By default, the *ActindoWorkflow Process Execution Failed* event is preselected.
+
```

Comment:
Link/Mention to Tasks module/Events menu entry?

Reply:
Comment left -> no "dead" link as long as task module documentation doesn't exist